grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java

```diff
@@ -165,54 +188,190 @@ private Json createTask(Request request, Response response) {
         String intervalParam = request.queryParams(REST.Request.TASK_RUN_INTERVAL_PARAMETER);
         String priorityParam = request.queryParams(REST.Request.TASK_PRIORITY_PARAMETER);
 
+        TaskState taskState = processTask(className, createdBy, runAtTime, intervalParam,
+                priorityParam);
+        manager.addTask(taskState, TaskConfiguration.of(bodyAsJson(request)));
+        // Configure the response
+        response.type(ContentType.APPLICATION_JSON.getMimeType());
+        response.status(200);
+
+        return Json.object("id", taskState.getId().getValue());
+    }
+
+    @POST
+    @Path("/bulk")
+    @ApiOperation(value = "Schedule a set of tasks.")
+    @ApiImplicitParams({
+            @ApiImplicitParam(name = REST.Request.TASKS_PARAM, value = "JSON Array containing an ordered list of tasks (see single task endpoint). The response references the order the task appears in the list as an index.", required = true, dataType = "List", paramType = "body"),
```

Comment:
Let me know if I have this wrong: While we can send multiple tasks to this endpoint, they must all have the same configuration–– we would not be able to send 10 loader tasks where each one is loading a different set of queries?